packages/cli/src/cmds/validator/keymanager/decryptKeystoreDefinitions/poolSize.ts

```diff
@@ -0,0 +1,16 @@
+let defaultPoolSize: number;
+
+try {
+  if (typeof navigator !== "undefined") {
+    defaultPoolSize = navigator.hardwareConcurrency ?? 4;
+  } else {
+    defaultPoolSize = (await import("node:os")).cpus().length;
```

Comment:
noting here that we should change this line to use [os.availableParallelism()](https://nodejs.org/api/os.html#osavailableparallelism) once we upgrade to node v20